WebApi/Models/BaseModel.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace WebApi.Models
+{
+    public class BaseModel
```

Comment:
abstract pois nunca iremos instanciar esta class, apenas herdar
```suggestion
    public abstract class BaseModel
```